build/package.json

```diff
@@ -33,7 +33,7 @@
     "rcedit": "~0.3.0",
     "request": "~2.27.0",
     "rimraf": "~2.2.2",
-    "runas": "^2",
+    "runas": "^3",
```

Comment:
^3.1 instead?  I know it doesn't make a difference, but the main package.json currently uses ^3.1 right now.